llarp/link/server.cpp

```diff
@@ -139,28 +140,60 @@ namespace llarp
           std::copy_n(buf.base, buf.sz, pkt.data());
           RecvFrom(from, std::move(pkt));
         });
-
+    std::string _ifname{ifname};
```

Comment:
If it needs an unconditional copy, might as well take it by value rather than `const&`.